manifests/init.pp

```diff
@@ -200,8 +200,8 @@
   String[1] $package_name                                          = $chrony::params::package_name,
   Optional[String] $package_source                                 = undef,
   Optional[String] $package_provider                               = undef,
-  $refclocks                                                       = [],
-  $peers                                                           = [],
+  Variant[Hash,Array[Variant[Hash,String[1]]]] $refclocks          = [],
```

Comment:
this is a confusing type…
we really need an example

Reply:
There is an example for `servers` higher up in the file. And the type for servers and
peers are the same.  Maybe we should just mention that?

```
#   This selects the servers to use for NTP peers. As with `servers` it can be an array of
#   servers or a hash of servers to their respective options.
```

Or perhaps we should move the example for servers to be below the description of peers and
servers?